Fastfile

```diff
@@ -164,14 +157,24 @@ def _get_team_id()
   team_id
 end
 
-def _parse_provision_file()
-  sh 'RUBYSCRIPT="$(echo \'def target(targetName, &doBlock)
-    profileName = doBlock.call
-    print targetName + "_" + "PROVISIONING_PROFILE = " + profileName + "\n"
+def _create_xcconfig(filename)
+  # TODO: Replace spaces with underscores?
+  # TODO: Try not to need `TAB_PROJECT_PATH`
+  project = Xcodeproj::Project.open(ENV['TAB_PROJECT_PATH'])
+  project.targets.each do |target|
+    profile = _get_profile_for_target(target)
+    if !profile.nil?
```

Comment:
This handily means test targets and anything else not in the export options plist are ignored.

Reply:
More readable as `if profile.present?`

Reply:
`present` isn't part of the standard Ruby library I believe, I tried this yesterday and I got an error on CI

Reply:
Unless I have to import something at the top of the file and it works automagically?

Reply:
Ah you're right, its added by Rails. My bad.